ci_framework/roles/build_containers/molecule/hotfix/converge.yml

```diff
@@ -0,0 +1,61 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_build_containers_cleanup: true
+    cifmw_build_containers_run_hotfix: true
+    cifmw_build_containers_hotfix_tag: patched
+    cifmw_build_containers_hotfix_images:
+      - quay.io/podified-master-centos9/openstack-keystone:current-podified
+  pre_tasks:
+    - name: Install dnf-plugins-core
+      ansible.builtin.package:
+        name: dnf-plugins-core
+        state: present
+      become: true
+
+    - name: Create /tmp/rpms directory
+      ansible.builtin.file:
+        path: /tmp/rpms
+        state: directory
+      become: true
+
+    - name: Download python3-requests RPM
+      ansible.builtin.command: sudo dnf download python3-requests --resolve
+      become: true
+      args:
+        chdir: /tmp/rpms
+
+    - name: Find RPM files in /tmp/rpms
+      ansible.builtin.find:
+        paths: /tmp/rpms
+        patterns: 'python3-requests-*.rpm'
+      register: rpm_files
+
+    - name: Set RPM path for cifmw_build_containers_hotfix_rpms_paths
+      ansible.builtin.set_fact:
+        cifmw_build_containers_hotfix_rpms_paths: "{{ rpm_files.files | map(attribute='path') | list }}"
+      when: cifmw_build_containers_run_hotfix | bool
+
+    - name: Set RPM path for cifmw_build_containers_hotfix_rpms_paths
+      ansible.builtin.set_fact:
+        cifmw_build_containers_hotfix_rpms_paths:
+          - "/tmp/rpms/python3-requests-*.rpm"
```

Comment:
At line 21: add `cifmw_build_containers_hotfix_rpms_paths: /tmp/rpms`  and remove this one also.